R/twas.R

```diff
@@ -328,7 +330,7 @@ twas_weights_cv <- function(X, Y, fold = NULL, sample_partitions = NULL, weight_
               args$data_driven_prior_matrices <- cv_args$data_driven_prior_matrices_cv[[j]]
             }
             if (method == "mvsusie_weights") {
-              args$prior_variance <- cv_args$data_driven_prior_matrices_cv[[j]]
+              args$prior_variance <- cv_args$mvsusie_data_driven_prior_matrices_cv[[j]]
```

Comment:
Can we call it `reweighted_data_driven_prior_matrices`? Don't call it `mvsusie_data_driven_prior_matrices_cv` so we know that in this input data the weight is refined.

Reply:
The name is updated in the new PR